utils/installer.py

```diff
@@ -258,17 +258,17 @@ def setup_libvirt():
 
 
 def setup_kubevirt():
-    master = deploy.kubevirt.master
-    master_username = deploy.kubevirt.master_user
-    master_password = deploy.kubevirt.master_passwd
+    master = deploy.kubevirt.endpoint
+    master_username = ''
```

Comment:
why do we still need the `master_username` param?

Reply:
It's not necessary, but let's remain them for the installer.py as an uniform format.